### ISSUE TITLE
"Note 2" appears twice in locales other than English

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/my-solar-system#179]
**Test device**
MacBook Air m1 chip

**Operating System**
13.3.1

**Browser**
safari/chrome

**Problem description**
For https://github.com/phetsims/qa/issues/935, on the Lab Screen "Note 2" appears twice in the info dialog--once in the translated language and again in English. 

**Visuals**
<img width="712" alt="Screenshot 2023-05-04 at 8 15 27 AM" src="https://user-images.githubusercontent.com/87318828/236202470-a523337b-f87a-4a07-9e6b-22fb388547f3.png">

<details>
Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Mi Sistema Solar‬
URL: https://phet-dev.colorado.edu/html/my-solar-system/1.2.0-rc.1/phet/my-solar-system_all_phet.html
Version: 1.2.0-rc.1 2023-05-02 15:04:18 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36
Language: en-US
Window: 1433x780
Pixel Ratio: 1.7999999523162842/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 31 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {}
</details>

**Comment by AgustinVallejo** [comment]
This is certainly weird, and probably more on the side of translation... My hypotheses is the following:
- The translatable string for the prototype probably contained both note 1 and 2.
- Prototype was published and translators got to it.
- Eventually we decided to add Note 3, and thus, we divided the original content string into content1, content2 and content3. content1 being the original string key for it.
- Now the sim has translation for content1 (Note1 and Note2) and also uses content2 (Note2) and content3 (Note3), so you can get "Note1translated Note2translated Note2 Note3".


I also noticed some weird cases where this happens:
![image](https://user-images.githubusercontent.com/37052186/236242262-cff72f96-be87-454f-a085-972a8ae93d4d.png)

Up to three Note2!! And some Note2 where the translation was not fully completed, which is concerning... Paging @arouinfar and @DianaTavares as I don't know how to tackle this issue.

**Comment by AgustinVallejo** [comment]
In today's stand up we decided to delete translations for the notes and have them be translated again. Should be done in pair with https://github.com/phetsims/my-solar-system/issues/87

**Comment by arouinfar** [comment]
> In today's stand up we decided to delete translations for the notes and have them be translated again.

Just for some additional context -- the strings have changed since the prototype, so deleting felt more appropriate than attempting to migrate

**Comment by zepumph** [comment]
@AgustinVallejo and I went through a dry run and things are working well. 

1. Wait until JUST BEFORE doing a production deploy. This way we don't have extra new translations sneak in in the next few days.
2. `cd GIT_ROOT; node ./rosetta/scripts/js/remove-string.js my-solar-system unitsInfo.content`
3. Reverted the changes to the en json file in my-solar-system.
4. `grunt update` in my-solar-system
5. inspect that only `unitsInfo.content` was deleted in all the babel files.
6. Commit to this issue.
